<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftDialog</class>
 <widget class="QDialog" name="leftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>left dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="title">
    <string>   zoom option</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="inButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="hello"/>
    </property>
    <property name="text">
     <string>zoom in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>zoom out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>shift up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>shift down</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3d">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3D particles</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_trace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>    trace
first particle</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_color">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>constant</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>initial position : L/R</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>diameter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>velocity magnitude</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V velocity : +/-</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>color by</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spin_plus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spin_minus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tilt_plus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tilt_minus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>spin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tilt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>create image file</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sphere_res">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fair</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>high</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
